src/lib/build.ts

```diff
@@ -281,9 +283,11 @@ export const build = async ({
     )
   );
 
-  startSpinner("Making sure all services are built");
-  await buildModules([...mapOfModuleConfigs.values()], marineCli);
-  stopSpinner();
+  if (serviceInfos.length > 0) {
+    startSpinner("Making sure all services are built");
+    await buildModules([...mapOfModuleConfigs.values()], marineCli);
+    stopSpinner();
```

Comment:
maybe put `stopSpinner` to `finally`? in case `buildModules` throws an error, wouldn't that cause spinner to spin indefinitely?

Reply:
Currently this function literally does nothing. I removed spinners because I tried a couple of them and they all just duplicate messages in some terminals, but a left the interface so in the future we can use some good spinner implementation that would work great

If the errror is thrown there - program will exit and spinner would have stopped together with that so there is nothing to worry about here

Reply:
So you're relying on a global error handling to stop the CLI on any error.

If we ever break that invariant, will have to go and add `try finally` in a lot of places.